src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.Docs.Build
 {
     internal static class TableOfContentsParser
     {
+        private static readonly string[] s_tocFileNames = new[] { "TOC.md", "TOC.json", "TOC.yml" };
+        private static readonly string[] s_experimentalTocFileNames = new[] { "TOC.experimental.md", "TOC.experimental.json", "TOC.experimental.yml" };
```

Comment:
>TOC.experimental.md [](start = 79, length = 19)

This is not supported in v2. It may wrongly treat a normal markdown as a experimental toc, whilc user hasn't a way to config it back to normal markdown.

Reply:
I will add experimental metadata in another PR

---
In reply to: [206503072](https://github.com/dotnet/docfx/pull/3127#discussion_r206503072) [](ancestors = 206503072,206031157)

Reply:
and i believe the toc metadata is another work for us to do. I don't believe current YAML/JSON TOC has supported metadata

---
In reply to: [206564281](https://github.com/dotnet/docfx/pull/3127#discussion_r206564281) [](ancestors = 206564281,206503072,206031157)

Reply:
I will open a issue for tracking

---
In reply to: [206565697](https://github.com/dotnet/docfx/pull/3127#discussion_r206565697) [](ancestors = 206565697,206564281,206503072,206031157)

Reply:
great!

---
In reply to: [206565770](https://github.com/dotnet/docfx/pull/3127#discussion_r206565770) [](ancestors = 206565770,206565697,206564281,206503072,206031157)